c/ckb.mol

```diff
@@ -45,14 +45,14 @@ table Script {
     args:           BytesVec,
 }
 
-struct OutPoint {
+table OutPoint {
```

Comment:
Those changes are actually introduced in this PR: https://github.com/nervosnetwork/ckb/pull/1505. The main PR in CKB hasn't been merged, but the PR for system script has. Maybe we should adapt the schema to the latest refactoring to avoid more changes?

Reply:
If adapt the change the script can not be used in my `chain_root` ckb PR. Seems the only solution is to wait for nervosnetwork/ckb#1505 .

I really think we should merge two PRs in the same time when these depends on each other, instead merge one first , then block all conflict PRs like this...

Reply:
Updated